.github/workflows/release.yml

```diff
@@ -95,6 +95,41 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           files: ./${{ env.OUTPUTDIR }}.tar.gz
 
+  docs-pull-request:
```

Comment:
Sorry, it would be a good feature, but would you send out this feature from this PR?

Reply:
Sure, but I need to modify the previous script before I can split it.

Reply:
> Sorry, it would be a good feature, but would you send out this feature from this PR?

https://github.com/containers/youki/pull/2537  @utam0k